apps/server/src/modules/tool/external-tool/uc/external-tool.uc.spec.ts

```diff
@@ -468,4 +481,77 @@ describe('ExternalToolUc', () => {
 			expect(externalToolService.deleteExternalTool).toHaveBeenCalledWith(toolId);
 		});
 	});
+
+	describe('getMetadataForExternalTool', () => {
+		describe('Authorization', () => {
+			const setupMetadata = () => {
+				const toolId: string = new ObjectId().toHexString();
+				return {
+					toolId,
+				};
+			};
+
+			it('should call getUserWithPermissions', async () => {
```

Comment:
we agreed to not call it "should call..." but the actual thing that it is trying to do.